app/pages/neworg/createorg/controller.test.js

```diff
@@ -120,14 +120,14 @@ describe('create org confirmation page controller', () => {
     });
 
     it('should return success as false with errors if api request is unsuccessful', async () => {
-      const mockError = { error: 'error' };
+      const mockError = { id: 'error' };
       apiProvider.postData
         .mockResolvedValueOnce({ data: { errors: [mockError] } });
 
       const response = await postAddOrg({ odsCode, data: { odsCode }, accessToken });
 
       expect(response.success).toEqual(false);
-      expect(response.errors).toEqual([mockError]);
+      expect(response.errorString).toEqual(mockError.error);
```

Comment:
`mock.error` will always be undefined? is this test working as expected

Reply:
oh yeah both parts are wrong 😄